client/app/lib/providers/resourcestatemodal/views/baseerrorpageview.coffee

```diff
@@ -33,18 +33,26 @@ module.exports = class BaseErrorPageView extends JView
         <cite>#{getCopyToClipboardShortcut()}</cite>
       """
 
+    _copied = no
     errorPartial = if isSingleError
     then _.escape errs.first
     else (errs.map (err) -> "<li>#{_.escape err}</li>").join ''
     @errorContent.addSubView new kd.CustomHTMLView
-      tagName  : if isSingleError then 'p' else 'ul'
+      tagName  : if isSingleError then 'pre' else 'ul'
       partial  : errorPartial
-      click    : -> copyToClipboard @getElement()
+      click    : ->
+        copyToClipboard @getElement()  unless _copied
+        _copied = yes
 
+    kd.utils.defer =>
+      @errorContainer.wrapper.scrollToBottom()
 
-  onPageDidShow: ->
 
+  onPageDidShow: -> kd.utils.defer ->
     # it needs to update container height if it can't be set fixed in css.
     # otherwise, custom scroll doesn't work properly
     container = @getDomElement().find '.main'
-    container.css 'height', container.height()
+    container.css 'height', container.height() or 400
```

Comment:
yes, `defer` in here solves the issue https://github.com/koding/koding/pull/9319#discussion-diff-84153652R51 but in any case I added that one as well if I missed something.